Explorer/Assets/DCL/Passport/Modules/IPassportModuleController.cs

```diff
@@ -0,0 +1,11 @@
+using DCL.Profiles;
+using System;
+
+namespace DCL.Passport.Modules
+{
+    public interface IPassportModuleController : IDisposable
```

Comment:
Yeah, I initially designed the passport modules with the interface because I saw that I was gonna manage them as a group from the main controller in some way, but I finally didn't took advantage of it.

As you suggested, all modules can be managed in a list an be able to setup and clear as a group. I applied the change here: https://github.com/decentraland/unity-explorer/pull/1277/commits/fc61f9e2805b306f7c2e8d6f5612b35f74f6dd67